infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LRRollingUpgradeHandler.java

```diff
@@ -81,5 +90,29 @@ public boolean isLRUpgradeInProgress(TxnContext txnContext) {
      */
     public void migrateData(TxnContext txnContext) {
         // Data migration to be added here.
+        eventSnapshotSyncRollingUpgrade(txnContext);
+    }
+
+    /**
+     * Add flag to event table to trigger snapshot sync.
+     */
+    public void eventSnapshotSyncRollingUpgrade(TxnContext txnContext) {
```

Comment:
How about:

eventSnapshotSyncRollingUpgrade -> triggerSnapshotSyncOnRollingUpgrade ?

Reply:
Since we are not actually triggering a snapshot sync here instead just enqueueing the event that will once processed maybe `addEventForSnapshotSyncOnRollingUpgrade()` is better?